features/roomlist/impl/src/main/kotlin/io/element/android/features/roomlist/impl/components/RoomListTopBar.kt

```diff
@@ -216,7 +202,7 @@ private fun DefaultRoomListTopBar(
         },
         navigationIcon = {
             if (matrixUser != null) {
-                IconButton(onClick = {}) {
+                IconButton(onClick = onOpenSettings) {
```

Comment:
We are loosing the `contentDescription`, this is not good for a11y. Is there a way to restore it? I would suggest to add it to the `Avatar` composable.

Reply:
Done in https://github.com/vector-im/element-x-android/pull/356/commits/9ecb6fd6f152fdbb393f24b48a020aa20931908c. Let's see if Maestro runs fine after this change.

Reply:
I tried it locally and it runs fine.